### ISSUE TITLE
[Visual requirement – Make Code – Home Page] Luminosity ratio is less than 3:1 for focus indicator present on all tiles under ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3547]
**User Experience:**
Visual impaired user will find difficulty to see the focus indicator over the control or element if that focus indicator have insufficient color contrast ratio as user won't be able to proceed further.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify contrast ratio of focus indicator over the all titles control like(New Project, Flashing Heart, Name tag, Smiley Buttons, Dice etc.)

**Actual Result:**
Luminosity ratio is less than 3:1 for focus indicator over the all titles control like(New Project, Flashing Heart, Name tag, Smiley Buttons, Dice etc.) under ‘Home page’ and due to this, visual impaired user is unable to see the focus indicator. 

**Expected Result:**
Luminosity ratio should be more than and equal to 3:1 for focus indicator over the all titles control like(New Project, Flashing Heart, Name tag, Smiley Buttons, Dice etc.) under ‘Home page’, so that visual impaired user could easily see the border of focus. Same issue exists on 'ARM template & Close' controls on same blade.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.4.11 - Non-text Contrast
](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: CCA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.11


![WCAG 1 4 11_CCA for focus indicator](https://user-images.githubusercontent.com/72871089/97849907-47beb300-1d19-11eb-8b20-011a64c89204.jpg)
, #Win10-Edge(Chromium)
